src/inc/eventtracebase.h

```diff
@@ -426,7 +523,7 @@ namespace ETW
         
         static VOID DomainLoad(BaseDomain *pDomain, __in_opt LPWSTR wszFriendlyName = NULL)
         {
-            if (MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_Context.IsEnabled)
+            if (ETW_PROVIDER_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER))
```

Comment:
Is ETW_PROVIDER_ENABLED specific to Windows, or are we going to implement this for other platforms, and just keep the name the same for convenience and PAL/VM naming.